com.ibm.wala.cast.python.test/data/tf2nn.py

```diff
@@ -0,0 +1,13 @@
+import tensorflow as tf
```

Comment:
Can you also add tests that use `@tf.function` like we did for the other?

Reply:
Addressed.